amt/locale/en_US/LC_MESSAGES/messages.po

```diff
@@ -178,6 +207,53 @@ msgstr ""
 msgid "Language"
 msgstr ""
 
+#: amt/site/templates/projects/details_base.html.j2:18
+msgid ""
+"Voldoet het algoritme aan de\n"
+"                            vereisten?"
+msgstr "Does the algorithm meet the requirements?"
+
+#: amt/site/templates/projects/details_base.html.j2:68
+#, fuzzy
+msgid "Ga naar alle vereisten"
+msgstr "Go to all requirements"
+
+#: amt/site/templates/projects/details_base.html.j2:75
+msgid "Welke instrumenten zijn uitgevoerd?"
+msgstr "Which instruments are executed?"
+
+#: amt/site/templates/projects/details_base.html.j2:78
+msgid "uitgevoerd"
```

Comment:
I think we agreed we need to have all the base keywords for translations in english. Also you can see that now this is not translated properly, because in the english page there are dutch words.

Reply:
Yes agree, I have changed it